src/utils/win/WinUtils.cpp

```diff
@@ -0,0 +1,30 @@
+#include "WinUtils.h"
+#include "windows.h"
+
+/////////////////////////////////////////////////////////////////////////////////////////
+bool WinUtils::getPowerManagementPrivileges()
```

Comment:
From what i have read, it should be, just that this priviledge is not a default one for processes.